packages/scripts/tests/unit/service/parameterService.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { findParameters } from '@/service/parameterService'
+
+describe('service/parameterService', () => {
+    describe('findParameters', () => {
+        it('Find non magma parameter', () => {
+            expect(findParameters('bla bla bla ${koe} bla bla', 'test')).toEqual(['koe'])
+        })
+    })
+
+    describe('findParameters', () => {
+        it('Find magma parameter', () => {
+            expect(findParameters("bla bla bla $('myValue') bla bla", 'JavaScript (Magma)')).toEqual(['myValue'])
```

Comment:
add (and support) a couple more interesting cases like `$("blah")` and `$('blah-di-blah')`

Reply:
See https://github.com/molgenis/molgenis/blob/master/molgenis-data-validation/src/main/java/org/molgenis/data/validation/meta/NameValidator.java for an improvement over `\w`